slatedb/clippy.toml

```diff
@@ -50,6 +50,8 @@ disallowed-methods = [
     { path = "rand::prelude::thread_rng", reason = "Use crate::rand::DbRand instead" },
     { path = "std::time::SystemTime::now", reason = "Use crate::clock::SystemClock instead" },
     { path = "std::time::SystemTime::elapsed", reason = "Use crate::clock::SystemClock instead" },
+    { path = "std::option::Option::unwrap", reasonon = "Use expect instead" },
```

Comment:
is `reasonon` correct? shouldn’t it be `reason`? I see a bunch of them, so we probably missed them before too.